charts/pega/templates/_pega-deployment.tpl

```diff
@@ -225,9 +225,9 @@ spec:
           # Maximum CPU and Memory that the containers for {{ .name }} can use
           # Resources are configured through deprecated settings. Use .tier[].resources instead
           limits:
-          {{- if .node.cpuLimit }}
+          {{- if and .node.cpuLimit (ne (include "deployWithoutCPULimit" (dict "node" .node)) "true") }}
```

Comment:
Can we simplify this in below sequence:-
If .node.cpuLimit is given, we always honour this.
else if we check for deploywithoutCPULimit
else we default it to 4 ?

Reply:
Also how  CMC customers will be aware of this feature? Is there plan to provide some documentation around it.